packages/web-components/src/components/page-header/page-header-breadcrumb.ts

```diff
@@ -23,7 +23,7 @@ class CDSPageHeaderBreadcrumb extends LitElement {
   /**
    * Specify if breadcrumb bar has bottom border.
    */
-  @property({ reflect: true })
+  @property({ attribute: 'border', type: Boolean })
```

Comment:
```suggestion
  @property({ reflect: true, type: Boolean })
```
No need to set the `attribute` property since the name is the same. We do need to have the `reflect` property set to true so that the `border` attribute can be picked up by the styles.